CHANGE(grafana): Provision 2 new diagnostic dashboards

fatpat opened the pull request:
##### SUMMARY
- diag_host_details: to view system details about one single host
- diag_hosts_comparison: to compare metrics between hosts

 ##### ISSUE TYPE
- Feature Pull Request

fatpat committed:
CHANGE(grafana): Provision 2 new diagnostic dashboards

 ##### SUMMARY
- diag_host_details: to view system details about one single host
- diag_hosts_comparison: to compare metrics between hosts

 ##### ISSUE TYPE
- Feature Pull Request

vdombrovski reviewed (changes requested):
Please import dashboards using provided tools/retriever.py script

fatpat committed:
CHANGE(grafana): Provision 2 new diagnostic dashboards

 ##### SUMMARY

 ##### ISSUE TYPE

 ##### SCOPE (skeleton only)

 ##### IMPACT

 ##### ADDITIONAL INFORMATION

fatpat commented:
> Please import dashboards using provided tools/retriever.py script

done

fatpat committed:
Remove 'editable' for diag dashboards

vdombrovski commented on files/diag_host_details.json:
```diff
@@ -0,0 +1,2178 @@
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.oioswift\"}) by (dimension)",
```
Hardcoded NS

vdombrovski commented on files/diag_host_details.json:
```diff
@@ -0,0 +1,2178 @@
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.meta1\"}) by (dimension)",
```
Hardcoded NS

vdombrovski commented on files/diag_host_details.json:
```diff
@@ -0,0 +1,2178 @@
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.rawx\"}) by (dimension)",
```
Hardcoded NS

vdombrovski commented on files/diag_host_details.json:
```diff
@@ -0,0 +1,2178 @@
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.meta2\"}) by (dimension)",
```
Hardcoded NS

vdombrovski commented on files/diag_host_details.json:
```diff
@@ -0,0 +1,2178 @@
+      "timeFrom": null,
+      "targets": [
+        {
+          "expr": "sum(netdata_apps_cpu_system_cpu_time___average{chart=\"apps.cpu_system\",family=\"cpu\",instance=~\"$host$\"}) by (dimension)",
```
Wrong percentage display (should be percent 0.0-1.0)

vdombrovski commented on files/diag_hosts_comparison.json:
```diff
@@ -0,0 +1,2190 @@
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(netdata_apps_cpu_system_cpu_time___average{chart=\"apps.cpu_system\",family=\"cpu\",instance=~\"$host$\"}) by (dimension)",
```
Wrong percentage display

vdombrovski commented:
Besides required fixes to merge, this PR only adds to dashboards to maintain without giving much more new information (all info can be found in other dashboards).

I am against integrating it into release until we don't have proper QA auto-validating dashboard queries.

cdelgehier commented:
ping @fatpat
